other/edge-server/client/react-client/src/components/Content.tsx

```diff
@@ -0,0 +1,131 @@
+import { useState } from "react";
+import {
+  Alert,
+  AppBar,
+  Button,
+  Card,
+  CardContent,
+  Container,
+  Toolbar,
+  Typography,
+} from "@mui/material";
+
+import { LoginPage } from "../pages/LoginPage";
+import { TodoItemsPage } from "../pages/TodoItemsPage";
+
+import { useAuthContext } from "../providers/AuthProvider";
+import { useAuth } from "../hooks/useAuth";
+
+import CheckCircleOutlineIcon from "@mui/icons-material/CheckCircleOutline";
+import ErrorOutlineIcon from "@mui/icons-material/ErrorOutline";
+
+import { EdgeConnectionStatus } from "../types";
+
+export function Content() {
+  const { logoutAndDisconnect } = useAuth();
+  const { user } = useAuthContext();
+
+  const [showError, setShowError] = useState(false);
+  const [showSuccess, setShowSuccess] = useState(false);
+  const [connectionMessage, setConnectionMessage] = useState("");
+  const [connectionError, setConnectionError] = useState("");
+  const [connectionString, setConnectionString] = useState("");
+
+  // TODO: abstract all of this one level up - to <Content>.
```

Comment:
Is this TODO still applicable? I'm inclined to say we should not merge something to the public repo that contains a "TODO" but I don't know if you've done this work and need to remove the comment, or if this work is still outstanding.

If it still needs to be done, maybe capture the info in a Jira ticket for future work and remove it from here?

Reply:
Oops, nope! I forgot to remove it. Thank you for catching!